mne/channels/tests/test_dig_montage.py

```diff
@@ -0,0 +1,95 @@
+# Author: Teon Brooks <teon.brooks@gmail.com>
+#
+# License: BSD (3-clause)
```

Comment:
If a class is in montage.py the test should be in test_montage.py. Please merge the 2 tests files for montages